src/service/api_client/__init__.py

```diff
@@ -0,0 +1,8 @@
+from core.config import IS_FAKE_API
+from service.api_client.hemas import ShemasAPIService
+from service.api_client.mock_shemas import MockShemasAPIService
+
+if IS_FAKE_API:
+    ConreateAPIService = MockShemasAPIService
+else:
+    ConreateAPIService = ShemasAPIService
```

Comment:
как то названия не очень, задаем там слово схемы. 
```suggestion
    APIService = MockAPIService
else:
    APIService = SiteAPIService
```

файлы с api сервисами тоже переименовать в mock_api_service и site_api_service

Reply:
А откуда взялось `Shemas`?? и что оно значит?
Гугл мне сказал, что шемас это какой-то чувак занимающийся вреслингом 
![image](https://user-images.githubusercontent.com/1431279/175832650-c71d2845-4833-4133-bff9-844fdc103aaf.png)

Если задумка была написать `Schemas`, то я ещё могу понять и то с трудом, так как это ни разу не упрощает понимание, а скорее наоборот.